app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  alias_attribute :author, :user
+  belongs_to :user, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_posts_counter
+
+  def update_posts_counter
+    user.posts_counter = user.posts.count
+  end
+
+  def recent_comments
+    comments.first(5)
```

Comment:
- [ ] The method `first` works for in memory data and can not be dependent on with database queries like in this case. Kindly use appropriate database queries that will give us reliable results. __HINT__: `comments.order('created_at Desc').limit(5)`